ui/app/common/concept-set/models/conceptSetSection.js

```diff
@@ -23,7 +23,7 @@ Bahmni.ConceptSet.ConceptSetSection = function (extensions, user, config, observ
         self.allowAddMore = config.allowAddMore;
         self.id = "concept-set-" + conceptSet.uuid;
         if (self.uuid === "1fb2dd86-53b5-4815-9c64-edc081b908d9") {
```

Comment:
@rubailly I asked Lisa for feedback on whether to collapse all forms or not. But if she decides we should collapse specific forms, I think we should use a global property to specify the list of forms that should be collapsed by default.